contracts/lib/Verifiers.sol

```diff
@@ -116,14 +116,16 @@ contract Verifiers is Assertions, SignatureVerification {
             return false;
         }
 
+        uint256 orderStatusNumerator = orderStatus.numerator;
```

Comment:
Comment here as well? Something like "Read order status numerator from storage and place on stack."